lib/msf/base/serializer/readable_text.rb

```diff
@@ -668,19 +668,34 @@ def self.dump_references(mod, indent = '')
     output
   end
 
-  # Dumps the aka names associated with the supplied module.
+  # Dumps the notes associated with the supplied module.
   #
   # @param mod [Msf::Module] the module.
   # @param indent [String] the indentation to use.
   # @return [String] the string form of the information.
-  def self.dump_aka(mod, indent = '')
+  def self.dump_notes(mod, indent = '')
     output = ''
 
-    if mod.notes['AKA'].present?
-      output << "AKA:\n"
-
-      mod.notes['AKA'].each do |aka_name|
-        output << indent + aka_name + "\n"
+    mod.notes.each do |name, val|
+      next unless val.present?
+
+      case name
+      when 'AKA'
+        output << "Also known as:\n"
+        val.each { |aka| output << "#{indent}#{aka}\n" }
+      when 'NOCVE'
```

Comment:
Note that 'NOCVE' and 'RELATED' were originally named all capital letters to match the first element of the 'References' array. If you wanted to recapitalize them to match the other things in Notes for consistency, that's fine too.

Reply:
I'll change `RELATED` back to what I had before: `RelatedModules`. I didn't want to realign my `info` hashes... :X

Reply:
I'm leaving `NOCVE` alone, since it's like an anti-`CVE` reference, and `AKA` reads better than `AlsoKnownAs`.